public/pages/utils/anomalyResultUtils.ts

```diff
@@ -594,6 +630,8 @@ export const parseBucketizedAnomalyResults = (result: any): Anomalies => {
             startTime: get(rawAnomaly, 'data_start_time'),
             endTime: get(rawAnomaly, 'data_end_time'),
             plotTime: get(rawAnomaly, 'data_end_time'),
+            attribution: featureBreakdownAttributionMap[get(feature, 'feature_id')],
```

Comment:
can you make `attribution`/`contribution` naming consistent? From my understanding they're being used interchangeably here.

Reply:
sure, will improve the naming